config/locales/de.yml

```diff
@@ -199,6 +223,28 @@ de:
         department_manager: Standort Betreuer
         name: Name
         user: User
+      group_offer:
+        all: Alle
+        availability: Zeitangaben
+        children: Kinder
+        date_time: Präzise Angaben (Tag und Uhrzeit) und genauen Zeitraum
+        description: Beschreibung des Angebotes
```

Comment:
Angebotes -> Angebots

Reply:
It is stated exactly like that in their document. Also: http://www.crodict.de/nomen/deutsch/Angebot.html

Reply:
Well, the customer is always right 😉